.github/workflows/cleanup.yml

```diff
@@ -18,6 +18,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
@MarcoSteinacher out of curiosity, why do you need that? I never had the need for it so far so I guess you managed to restrict the default permissions?